LogMonitor/src/LogMonitor/FileMonitor/Utilities.cpp

```diff
@@ -0,0 +1,162 @@
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+#include "pch.h"
+
+/**
+ * Warapper around Create Event API
+ * 
+ * @param bManualReset
```

Comment:
I see popular C++ guides are silent on how do the function block comment but I like this one, looks clean.